thoughts-of-a-rustacean-learning-go.md

```diff
@@ -0,0 +1,244 @@
+<!--
+headline: Thoughts of a Rustacean learning Go
+description: the experiences of a Rustacean learning Go
+author: Manish Goregaokar
+datePublished: 2015-04-24
+twitter: ManishEarth
+github: manishearth
+-->
+
+// Might want to mention that this is a crosspost
+
+
+For the past year or so I've been playing with the [Rust programming language][trpl].
+I've come to love the language and spend quite a bit of my free time contributing to the [compiler][compiler]
+and the [Servo browser engine][servo].
+
+// Perhaps a different form of introduction?
```

Comment:
I like this introduction, it's neat to get some information about your background in Rust.